### ISSUE TITLE
Fix attachments encoding with apostrophe in file name

### ISSUE THREAD

**Issue by sosnovsky** [FlowCrypt/flowcrypt-browser#5150]
Looks like google doesn't like this apostrophe in the middle of the file name (between `what` and `s_up`
```
Content-Disposition: attachment;
 filename*0*=utf-8''what's_up%25253F.txt.pgp
```
Probably this is why extra double quotes appear
![image](https://github.com/FlowCrypt/flowcrypt-browser/assets/23335944/3a8d0aba-3409-458a-8058-bcdb2ff3a2cd)
```
        {
          "partId": "1",
          "mimeType": "application/octet-stream",
          "filename": "\"what\"'s_up%25253F.txt.pgp",
```

According to https://datatracker.ietf.org/doc/html/rfc2231#section-3
the apostrophe has special meaning like in this example
```
 title*0*=us-ascii'en'This%20is%20even%20more%20
```
we have the 2 required apostrophes right after `utf-8` denoting unspecified language, that's ok, but we should probably escape the apostrophe with %27, so we can take the filename directly from Gmail API (when decoding).
If the filename was encoded like this by the browser extension, we should also fix the encoding part.
@sosnovsky ?

_Originally posted by @rrrooommmaaa in https://github.com/FlowCrypt/flowcrypt-browser/pull/4537#discussion_r1192634117_

**Comment by sosnovsky** [comment]
To reproduce - try to send encrypted message with attachment named `what's_up?.txt`

**Comment by ioanmo226** [comment]
@sosnovsky I can see that all tasks in [8.4.8](https://github.com/FlowCrypt/flowcrypt-browser/milestone/474) milestone are finished.
Maybe we can release this version?
Or you are waiting something?

**Comment by sosnovsky** [comment]
> @sosnovsky, I noticed that all the tasks in the 8.4.8 milestone have been completed. Should we consider releasing this version? Or is there something specific you are waiting for?

No, I think everything is ready for 8.4.8 release, it's already includes a lot of fixes.
Tom usually creates new builds and releases them.

@tomholub should we release new 8.4.8 version?
